backend/app/crud/transaction.py

```diff
@@ -1,13 +1,33 @@
+from sqlalchemy import func
 from sqlalchemy.orm import Session
 
 from app.models import TransactionORM
-from app.schemas import Transaction
+from app.schemas import Transaction, Balance
 
 
-def get_transactions(db: Session, skip: int = 0, limit: int = 100):
+def get_balance(db: Session):
```

Comment:
Maybe leave a comment, that we mock the balance with the last amount of a transaction.